src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let totalPrice = PRICE_PER_DAY * days;
+
+  if (days >= LONG_TERM) {
+    totalPrice -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    totalPrice -= SHORT_TERM_DISCOUNT;
+  }
```

Comment:
let's make totalPrice a `const`
and use `if..return` construct to avoid `else if`